src/main/java/club/sk1er/autogg/AutoGG.java

```diff
@@ -52,46 +49,48 @@ public void init(FMLInitializationEvent event) {
         autoGGConfig = new AutoGGConfig();
         autoGGConfig.preload();
 
-        running = false;
+        AutoGGListener listener = new AutoGGListener();
```

Comment:
why is this being hoisted?

Reply:
wanted to test a non-static version but scrapped it and forgot to remove that